app/src/main/java/ch/epfl/skysync/screens/ConfirmFlightScreen.kt

```diff
@@ -0,0 +1,35 @@
+package ch.epfl.skysync.screens
+
+import androidx.compose.runtime.Composable
+import androidx.navigation.NavController
+import ch.epfl.skysync.components.confirmationScreen
+import ch.epfl.skysync.models.calendar.TimeSlot
+import ch.epfl.skysync.models.flight.Balloon
+import ch.epfl.skysync.models.flight.BalloonQualification
+import ch.epfl.skysync.models.flight.Basket
+import ch.epfl.skysync.models.flight.FlightType
+import ch.epfl.skysync.models.flight.PlannedFlight
+import ch.epfl.skysync.models.flight.Role
+import ch.epfl.skysync.models.flight.RoleType
+import ch.epfl.skysync.models.flight.Team
+import ch.epfl.skysync.models.flight.Vehicle
+import java.time.LocalDate
+
+@Composable
+fun confirmationScreenHardCoded(navController: NavController) {
```

Comment:
you can actually already (almost) fully connect your screen to the navigation. Check out how the ViewModel is init. for the`FlightDetailsScreen`. Your toplevel composable should take the arguments: 

```
navCtr: NavHostController,
flightId:String
vm: FlighstViewModel
```
Inside your toplevel function you ignore the flightId and viewModel for now and use the dummyFlight as is as well as the dummy onConfirmationClick

Reply:
Done